kitchen.appveyor.yml

```diff
@@ -0,0 +1,118 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_zero
+  always_update_cookbooks: true
+  product_name: chef
+  product_version: 15.2.20
+  install_strategy: once
+  attributes:
+    se_baseline:
+      chef_client_version: '15.2.20'
+      configure_monitoring: false
+    nix_baseline:
+      chef_client_version: '15.2.20'
```

Comment:
Is this required for both the se_baseline and the nix_baseline?

Reply:
No...Unless I sometimes use my personal baseline:)